sites/docs/docs/markdown/index.md

```diff
@@ -135,6 +135,10 @@ To display data in a table, use a [Data Table](../components/data-table) instead
 
 ## Frontmatter
 
+:::note
+Frontmatter does not support Javascript statements at this time; and things may behave unexpectedly if wrapped in `{}`
+:::
+
```

Comment:
Do you have a take on whether it would be a good idea to do this in the future? Or whether it could open a can of worms?

Unless we are reasonably sure we'll end up supporting it, I'd consider:

Frontmatter does not support Javascript - Expressions wrapped in `{}` may behave unexpectedly.